docs/source/conf.py

```diff
@@ -71,7 +71,7 @@
 # This pattern also affects html_static_path and html_extra_path.
 html_extra_path = ["robots.txt"]
 
-exclude_patterns: list = []
+exclude_patterns: list[str] = ["reference/manim.mobject.opengl**"]
```

Comment:
On my machine sphinx was building the docs for OpenGL as well, which failed due to `OpenglVGroup` not being defined (I have no idea why that was happening) so excluded them from the build using this switch and it worked after that.

Docs [here](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-exclude_patterns)

Reply:
Does it fail on RTD? If not I think we can undo this change and you should file an issue. And even if it does fail, it's better to split it up into a new PR for cleaner commit history, since this seems unrelated.

Reply:
Ok shall revert